builder/lxc/step_export.go

```diff
@@ -108,6 +129,31 @@ func (s *stepExport) Run(state multistep.StateBag) multistep.StepAction {
 	return multistep.ActionContinue
 }
 
+func (s *stepExport) PrepareExport(containerDir string, exportFolders []ExportFolder) (error, string) {
+	containerDir = filepath.Join(containerDir, "rootfs")
+	exportFolder := filepath.Join(containerDir, "lxc-export-container-dir")
+	err := s.SudoCommand([]string{ "mkdir", "-p", exportFolder}...)
+	if err != nil {
+		return nil, exportFolder
+	}
+	for i := 0; i < len(exportFolders); i++ {
+		src := filepath.Join(containerDir, exportFolders[i].Src)
+		dest := filepath.Join(exportFolder, exportFolders[i].Dest)
+		destFolder := filepath.Dir(dest)
+		if destFolder != exportFolder {
+			err := s.SudoCommand([]string{ "mkdir", "-p", destFolder}...)
```

Comment:
I *think* (not sure, not a Go person) that `s.SudoCommand([]string{"mkdir", "-p", destFolder}...)` can be done without destructuring: `s.SudoCommand("mkdir", "-p", destFolder)`